### PR TITLE
Fix TensorFLow example

### PR DESCRIPTION
**Pull request by JanuszL**
- fix type in TensorFlow + DALI pipeline example
- make L3 test to run DALI TF example instead of container one

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 624048

File: docs/examples/tensorflow/demo/nvutils/image_processing.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def __init__(self,
 
         if training:
             if dali_cpu:
-                self.resize = dali.ops.Resize (device="cpu", resize_x=width, resize_y=heigth)
+                self.resize = dali.ops.Resize (device="cpu", resize_x=width, resize_y=height)
             else:
                 self.resize = dali.ops.RandomResizedCrop(
                     device=resize_device,
```
Comment (Kh4L) [inline docs/examples/tensorflow/demo/nvutils/image_processing.py]: Actually `resize_device` is not set anymore (after @jantonguirao 's changes).
Please set it as [it was set prior to the changes](https://github.com/NVIDIA/DALI/blob/b76919292f808a48d69aa9ac1d518bde51dbf274/docs/examples/tensorflow/demo/nvutils/image_processing.py#L64), or remove it

Comment (JanuszL) [reply docs/examples/tensorflow/demo/nvutils/image_processing.py]: Done

**Comment by JanuszL** [comment]
Rebuilds 624203

**Pull request merged**